### PR TITLE
#2420 - Do not create a game with empty title

### LINKED ISSUE #2420: Do not create a game with empty title using only whitespace
In the "Custom Games" tab, I saw the game with the empty title. The title contains only a whitespace.

Client Version:
2021.10.0

image

### PR DESCRIPTION
**Pull request by Marc-Spector**
Closes #2420

### CODE REVIEW

File: src/test/java/com/faforever/client/game/CreateGameControllerTest.java [hunk 1]
```diff
@@ -221,18 +221,18 @@ public void testButtonBindingIfFeaturedModNotSet() {
...
-
+    String message = "title missing";
+    when(i18n.get("game.create.titleMissing")).thenReturn(message);
+    runOnFxThreadAndWait(() -> instance.initialize());
     assertThat(instance.titleTextField.getText(), is(""));
-    assertThat(instance.createGameButton.getText(), is("title missing"));
+    assertThat(instance.createGameButton.getText(), is(message));
+
+    runOnFxThreadAndWait(() -> instance.titleTextField.setText(" "));
+    assertThat(instance.createGameButton.getText(), is(message));
```
Comment (Sheikah45) [inline src/test/java/com/faforever/client/game/CreateGameControllerTest.java:232]: Should assert that the text has changed to " " as well

**Comment by Marc-Spector** [comment]
`testOnReport` method of `ReportDialogControllerTest` class was failed. This method was successful on my computer. I will try to fix it.

**Comment by Sheikah45** [comment]
Rebase onto develop and the error should be fixed now by https://github.com/FAForever/downlords-faf-client/pull/2427

**Comment by Sheikah45** [comment]
I went ahead and rebased it

**Review by Sheikah45 — APPROVED** [review]

**Comment by Marc-Spector** [comment]
@Sheikah45 Great! Thanks for the fix.

**Pull request merged**